addons/project/models/project.py

```diff
@@ -761,6 +762,11 @@ def project_update_all_action(self):
         action['display_name'] = _("%(name)s's Updates", name=self.name)
         return action
 
+    def action_project_sharing(self):
```

Comment:
We could add an ensure_one here
```suggestion
    def action_project_sharing(self):
        self.ensure_one()
```